prac_07/myguitars.py

```diff
@@ -0,0 +1,20 @@
+from prac_06.guitar import Guitar
+import csv
+
+
+def main():
+    guitars = []
+
+    in_file = open('guitars.csv', 'r')
+    for line in in_file:
+        parts = line.strip().split(',')
+        guitar = Guitar(parts[0], int(parts[1]), float(parts[2]))
+        guitars.append(guitar)
+    in_file.close()
```

Comment:
this is good however you can do this instead to eliminate the need to use in_file.close()

    with open(FILENAME, "r") as in_file:
        for line in in_file:
            guitars.sort()
            parts = line.strip().split(',')
            guitar = Guitar(parts[0], parts[1], parts[2])
            guitars.append(guitar)